libc/cmake/modules/CheckCompilerFeatures.cmake

```diff
@@ -57,3 +57,5 @@ foreach(feature IN LISTS ALL_COMPILER_FEATURES)
 endforeach()
 
 message(STATUS "Compiler features available: ${AVAILABLE_COMPILER_FEATURES}")
+
+check_cxx_compiler_flag("-nostdlib++" LIBC_COMPILER_SUPPORT_NOSTDLIBPP)
```

Comment:
will conflict with #78776

Reply:
Can you use a similar naming convention/prefix `LIBC_CC_SUPPORTS_`

Reply:
Updated the flag name.  I can wait for you to merge #78776 first.

Reply:
> can wait for you to merge https://github.com/llvm/llvm-project/pull/78776 first.

done